Tasks/NpmAuthenticateV0/package.json

```diff
@@ -17,7 +17,13 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "2.8.0",
-    "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz"
```

Comment:
Why is the packaging-common being replaced with azure-pipelines-task-packing-common?